joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

l1_controller:
  type: "position_controllers/JointPositionController"
  joint: l0_to_l1

l2_controller:
  type: "position_controllers/JointPositionController"
  joint: l1_to_l2

l3_controller:
  type: "position_controllers/JointPositionController"
  joint: l2_to_l3

l4_controller:
  type: "position_controllers/JointPositionController"
  joint: l3_to_l4

l5_controller:
  type: "position_controllers/JointPositionController"
  joint: l4_to_l5

l6_controller:
  type: "position_controllers/JointPositionController"
  joint: l5_to_l6

l7_controller:
  type: "position_controllers/JointPositionController"
  joint: l6_to_l7

